### PR TITLE
Update seed database to 2.3.1

### PR DESCRIPTION
**Pull request by sandertan**
- Update human and mouse seed database to 2.3.1 (includes tables for gene sets)
- The PDB schema has not changed, so only the file name is updated
- Updated documentation to include new section with previous seed database(s), linking to commit hashes.

This addresses https://github.com/cBioPortal/datahub/issues/66

### CODE REVIEW

File: seedDB/README.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+- 1.5.1
+- 1.5.2
+
+Schema 2.1.0: [SQL file with create table statements for portal release 1.5.0](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.5.1/db-scripts/src/main/resources/cgds.sql)<br>
+Seed database part1: [cbioportal-seed SQL (.gz) file - part1 (no PDB tables)](https://github.com/cBioPortal/datahub/raw/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/seed-cbioportal_hg19_v2.1.0.sql.gz)<br>
+Seed database part2 (optional): [cbioportal-seed SQL (.gz) file - part2 (only PDB tables)](https://github.com/cBioPortal/datahub/raw/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/seed-cbioportal_hg19_v2.1.0_only-pdb.sql.gz)
+
+When using a release version > 1.5.2, a migration step to a new database schema is required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
+
+## Additional information
```
Comment (oplantalech) [inline seedDB/README.md]: I think this section should be removed, and the content should go before the section "Latest seed database"

File: seedDB_mouse/README.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
+- 1.5.0
+- 1.5.1
+- 1.5.2
+
+Schema 2.1.0: [SQL file with create table statements for portal release 1.5.0](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.5.1/db-scripts/src/main/resources/cgds.sql)<br>
+Seed database: [cbioportal-seed SQL (.gz) file)](https://github.com/cBioPortal/datahub/raw/8031b659b99c833d7fbcd057834220cd6708a032/seedDB_mouse/seed-cbioportal_mm10_v2.1.0.sql.gz)<br>
+
+When using a release version > 1.5.2, a migration step to a new database schema is required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
+
+## Additional information
```
Comment (oplantalech) [inline seedDB_mouse/README.md]: I think this section should be removed, and the content should go before the section "Latest seed database"

File: seedDB/README.md [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+
+Schema 2.1.0: [SQL file with create table statements for portal release 1.5.0](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.5.1/db-scripts/src/main/resources/cgds.sql)<br>
+Seed database part1: [cbioportal-seed SQL (.gz) file - part1 (no PDB tables)](https://github.com/cBioPortal/datahub/raw/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/seed-cbioportal_hg19_v2.1.0.sql.gz)<br>
+Seed database part2 (optional): [cbioportal-seed SQL (.gz) file - part2 (only PDB tables)](https://github.com/cBioPortal/datahub/raw/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/seed-cbioportal_hg19_v2.1.0_only-pdb.sql.gz)
+
+## Additional information
+The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database 2.1.0. This means that the version numbers won't be identical. 
+
+**For developers:** Updating the seed database for Datahub is described [here](Update-Seed-Database.md)
```
Comment (oplantalech) [inline seedDB/README.md]: I think this section should be removed, and the content should go before the section "Latest seed database"

File: seedDB_mouse/README.md [hunk 4]
```diff
@@ -1,18 +1,34 @@
...
+- 1.5.0
+- 1.5.1
+- 1.5.2
+
+When using this older seed database with a release version > 1.5.2, a migration step to a new database schema is required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
+
+Schema 2.1.0: [SQL file with create table statements for portal release 1.5.0](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.5.1/db-scripts/src/main/resources/cgds.sql)<br>
+Seed database: [cbioportal-seed SQL (.gz) file)](https://github.com/cBioPortal/datahub/raw/8031b659b99c833d7fbcd057834220cd6708a032/seedDB_mouse/seed-cbioportal_mm10_v2.1.0.sql.gz)
+
+## Additional information
```
Comment (oplantalech) [inline seedDB_mouse/README.md]: I think this section should be removed, and the content should go before the section "Latest seed database"

**Comment by sandertan** [comment]
Implemented feedback on documentation.

**Pull request merged**